Updated parquet version and fixed dependencies to use new features li…

ajitgogul opened the pull request:
…ke rowcount and size

ajitgogul commented:
Mainly aimed to make use of the rowcount and size feature introduced in parquet 1.10.0 : https://issues.apache.org/jira/browse/PARQUET-196

regadas commented on .travis.yml:
```diff
@@ -1,7 +1,7 @@
 scala:
   - 2.11.12
-jdk: oraclejdk7
+jdk: oraclejdk9
```
For this part can you add a matrix?

ajitgogul replied:
fixed.

regadas commented on build.sbt:
```diff
@@ -2,10 +2,10 @@ organization := "com.spotify.data"
-val avroVersion = "1.8.2"
-val parquetVersion = "1.9.0"
+val avroVersion = "1.9.0"
+val parquetVersion = "1.10.1"
```
we can even possibly bump this to 1.11.0

ajitgogul replied:
sure.

ajitgogul replied:
for some reason, sbt is not able to find 1.11.0

regadas commented on build.sbt:
```diff
@@ -2,10 +2,10 @@ organization := "com.spotify.data"
 val hadoopVersion = "2.7.4"
-val avroVersion = "1.8.2"
-val parquetVersion = "1.9.0"
+val avroVersion = "1.9.0"
```
let's not bump avro in this PR, parquet up to 1.11.0 uses 1.8.2

avro 1.9 as some breaking changes

ajitgogul replied:
sure. thanks for the review. will update.

ajitgogul commented:
- Bumped parquet version. Avro & Protobuf remain unchanged.
- Travis build was failing for oraclejdk7. So updated to matrix with oraclejdk9.
- sbt build version < 1.x had issues with oraclejdk9. updated build version.
- Updated scala version in build and travis.yml
- Updated README with the correct versions for parquet, avro and protobuf
- Fixed build.sbt syntax as per sbt 1.x as the older one was no longer supported
